### PR TITLE
WIP Ana/loop to get new networks from db

### LINKED ISSUE #4645: Add new network when set to enabled
I just realized while attempting to recover the akash-testnet network that, apparently, we don't have the logic there to add a new network from DB when setting it to enabled.

This should be fairly easy and we could include the logic for it in the same loop where we watch for other network changes, I think. It should be quite easy to do, in any case

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4645 

**Description:**

Looks good but sadly doesn't work. Nothing is that easy. There are no one line changes :shrug: 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Currently it works adding networks that become `enabled` in DB, but not so when you change them to `disabled`

Any idea about this @faboweb ? Maybe I need to "kill" the app object and redefine it?

File: api/index.js [hunk 1]
```diff
@@ -58,8 +58,9 @@ async function pollForNewNetworks(httpServer, app) {
     )
     console.log(`Transaction service ready at ${config.transactionPath}`)
   })
-  this.newNetworksPollingTimeout = setTimeout(async () => {
-    this.pollForNewNetworks()
+  newNetworksPollingTimeout = setTimeout(async () => {
+    httpServer.close({ port: config.port })
```
Comment (faboweb) [inline api/index.js:62]: what?!

**Comment by faboweb** [comment]
I think the approach is wrong: I would query all networks and update all networks always. Then handle polling in the subscriptions based on the enabled state.

**Comment by Bitcoinera** [comment]
> Then handle polling in the subscriptions based on the enabled state.

But the subscriptions are created only afterwards, per each filtered network. That is why you need to do this even before. Or am I missing something here?

File: api/index.js (hunk 1 above)
Comment (Bitcoinera) [reply api/index.js:62]: I was getting an error here. You cannot do `httpServer.listen` without closing the port before, once opened

**Comment by Bitcoinera** [comment]
Closing this as the logic to accomplish it would imply more serious changes, and with `enabled` and `experimental` we already have enough tools to play with

**Pull request closed without merging**